src/components/NavPageLink.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string;
+  text: string;
+};
+
+export const NavPageLink:FC<Props> = ({ to, text }) => {
+  return (
+    <NavLink
+      to={to}
+      className={({ isActive }) => classNames('navbar-item',
```

Comment:
It's better to use 'isActive' directly in the 'classNames' function to improve readability. Example: classNames('navbar-item', { 'has-background-grey-lighter': isActive })